lib/assign-properties.js

```diff
@@ -43,13 +44,18 @@ if (sourceAvailable) {
 function assignProperties(target, options) {
   var value;
 
-  for (var key in options) {
-    value = options[key];
-
-    if (typeof value === 'function' && hasSuper(value)) {
-      target[key] = giveMethodSuper(target, key, value);
-    } else {
-      target[key] = value;
+  if (typeof target === 'function') {
```

Comment:
Will remove. Was branching so I didn't break the shit out of the unit tests when I was making changes.